### ISSUE TITLE
Design: make location preview and location details view nice-looking

### ISSUE THREAD

**Issue by SebuLeugger** [veganaut/veganaut#140]
The location preview could look nicer than it does atm. Also, I'm not sure whether we need the teams' scores in the preview, or whether the "check out location" button needs to be so big, or needs so much text (it could just say "more", couldn't it?). Also, even though we just changed that, I now think it might still be better to add text to the location type symbol, i.e. either "gastronomy" or "shopping". Or maybe not. Anyway, it just has to look nicer. 

The same goes for the top part of the location details view. The elements may have to be re-arranged or have their appearance changed, especially the location type symbol and the teams' scores.

**Comment by SebuLeugger** [comment]
We decided yesterday to definitely delete the team's scores in the preview. For users who aren't logged in, we don't replace it with anything. For users who are logged in we replace it by a single number (maybe in combination with some not-yet-defined symbol) which shows how many points your team needs to conquer the location / how many points your team is ahead of the runner-up team.

Also, we kind of decided to combine the offerQuality and effortValue symbols into one bigger symbol, with the effortValue symbol (the weather icon) ABOVE the offerQuality symbol (the plant icon). This bigger symbol would span two lines. We made some sketches on a piece of paper of how it might look like.
